docs/UserGuide.md

```diff
@@ -153,183 +142,158 @@ Format: `vedit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
 * Existing values will be updated to the input values.
 * When editing skills, the existing skills of the volunteer will be removed i.e adding of skills is not cumulative.
 * You can remove all the volunteer’s skills by typing `t/` without
-    specifying any skills after it.
+  specifying any skills after it.
 
 Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st volunteer to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd volunteer to be `Betsy Crower` and clears all existing skills.
 
-### Locating volunteers by name: `vfind`
-
-Finds volunteers whose names contain any of the given keywords.
-
-Format: `vfind KEYWORD [MORE_KEYWORDS]`
-
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Volunteers matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+### Deleting a volunteer from the volunteer list: `vdelete`
 
-Examples:
-* `vfind John` returns `john` and `John Doe`
-* `vfind alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
-
-### Deleting a volunteer : `vdelete`
-
-Deletes the specified volunteer from the volunteer list.
+Volunteer coordinators can delete volunteers and remove them from the volunteer list if they no longer wish to volunteer anymore.
 
-Format: `vdelete INDEX`
+Format: `vdelete VOLUNTEER_ID`
 
 * Deletes the volunteer at the specified `INDEX`.
 * The index refers to the index number shown in the displayed volunteer list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* The index **must be a positive integer** 1, 2, 3, …
+
+Restrictions:
+* The volunteer ID must be an integer that represents a valid volunteer number in the volunteer list. If there are 30 volunteers in the volunteer list, the acceptable values will be from 1-30.
```

Comment:
i think to be more precise instead of volunteer list we could say the list of volunteers currently displayed to the user? it's a small issue tho

Reply:
agree cos vfind is already there